app/assets/javascripts/sorry.js

```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
```

Comment:
I know this was auto-generated, but would you mind removing it since there is no actual content?